### PR TITLE
gerundizes recently-added titles in the learner guides

### PR DESCRIPTION
**Pull request by lamagnifica**
@srpearce and @catong thumbs or other suggestions?

### CODE REVIEW

File: en_us/shared/students/SFD_self_paced.rst [hunk 1]
```diff
@@ -1,12 +1,12 @@
 .. _SFD Self Paced:
 
-##################
-Self-Paced Courses
-##################
+###########################
+Taking a Self-Paced Course
+###########################
```
Comment (catong) [inline en_us/shared/students/SFD_self_paced.rst:5]: The gerundizing is good. Maybe we can decide on whether to use singular or plural in headings. We have singular in this heading, but plural in the teams topic below. I strongly believe plural makes more sense in headings for conceptual topics; singular always strikes me as unnecessarily specific.

**Comment by srpearce** [comment]
I'm with Carol re: singular vs. plural.

**Comment by lamagnifica** [comment]
travis test this

**Comment by lamagnifica** [comment]
tests pass locally

**Pull request merged**